RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -156,11 +282,71 @@ namespace {
   }
 }  // namespace
 
-double DAClusterizerInZT_vect::update(
-    double beta, track_t& gtracks, vertex_t& gvertices, bool useRho0, const double& rho0) const {
+void DAClusterizerInZT_vect::set_vtx_range(double beta, track_t& gtracks, vertex_t& gvertices) const {
+  const unsigned int nv = gvertices.getSize();
```

Comment:
nv=0 is not possible here. There is only one situation in which the cluster list can be empty, which is - temporarily - when the first and only cluster is split in ZT clustering, because there, the existing cluster is removed before two new ones are created. But set_vtx_range isis called only after adding the two new clusters. Otherwise vertices are only added or merged, never just removed. But I an add the check anyway or an assertion with #ifdef debug

Reply:
protection added